packages/lib/src/components/GooglePay/utils.ts

```diff
@@ -11,3 +11,15 @@ export function resolveEnvironment(env = 'TEST'): google.payments.api.Environmen
             return 'TEST';
     }
 }
+
+export function mapBrands(brands) {
+    const brandMapping = {
+        mc: 'MASTERCARD',
+        amex: 'AMEX',
+        visa: 'VISA',
+        interac: 'INTERAC',
+        discover: 'DISCOVER'
+    };
+    const allowedCardNetworks = brands.map(brand => brandMapping[brand]);
+    return allowedCardNetworks.reduce((unique, item) => (unique.includes(item) ? unique : [...unique, item]), []);
```

Comment:
About the reducer to remove duplicated items, why would it be possible for items to be duplicated?

Reply:
yes, it is possible, because, we have elodebit and elo txvariants, however applepay has only txvariant named elo. Hence when we map this case, elo could be duplicated.

Reply:
So we could check for duplicated items only for Apple Pay then?